token/cli/src/main.rs

```diff
@@ -614,6 +614,7 @@ async fn command_authorize(
         AuthorityType::TransferFeeConfig => "transfer fee authority",
         AuthorityType::WithheldWithdraw => "withdraw withheld authority",
         AuthorityType::InterestRate => "interest rate authority",
+        AuthorityType::PermanentDelegate => "permanent delegate",
```

Comment:
Should this be
```suggestion
        AuthorityType::PermanentDelegate => "permanent delegate authority",
```

Reply:
imo no, existing convention on token objects is `freeze_authority` `close_authority` etc, but jsut plain `delegate`. in my head, if a "delegate authority" existed, it would be something that can modify the delegate